modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -195,6 +200,54 @@ interfaces:
       auto-dns: true
 # ...
 ----
+<1> If you are using this configuration with dnsmasq or another DNS caching system, consider using `127.0.0.1`.
+
+* Configure a status DNS search option using Dynamic Host Configuration Protocol (DHCP) and IPv6 Stateless Address AutoConfiguration (SLAAC) servers.
++
+.Static DNS search option with DHCP/autoconf name servers
+[source,yaml]
+----
+dns-resolver:
+  config:
+    search:
+      - example.com
+      - example.org
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+    ipv6:
+      enabled: true
+      dhcp: true
+      autoconf: true
+# ...
+----
+
+* Configure a static DNS for a network interface with an automatic IP configuration. Note that when updating the `/etc/resolv.conf` file on a host node, you must not specify an interface (IPv4 or IPv6) in the `NodeNetworkConfigurationPolicy` (NNCP) manifest.
++
+.Static DNS for network interface with automatic IP configuration
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+ name: worker-0-dns-testing
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: <target_node>
+dns-resolver:
+  config:
+    search:
+    - example.com
+    - example.org
+    server:
+    - 2001:db8:f::1
+    - 192.0.2.251
+# ...
+----
```

Comment:
At the beginning of the DNS example section, it said 'By default, the nmstate API stores DNS values globally...', and the example in Line232-250 is for the default situation, so how about move this example to the beginning of DNS section?

Reply:
Moved to beginning.